ext/sqlite3/sqlite3.stub.php

```diff
@@ -86,10 +86,10 @@ class SQLite3Stmt
     private function __construct(SQLite3 $sqlite3, string $sql) {}
 
     /** @return bool */
-    public function bindParam(string|int $param_number, mixed &$param, int $type = SQLITE3_TEXT) {}
+    public function bindParam(string|int $param, mixed &$bind_var, int $type = SQLITE3_TEXT) {}
```

Comment:
TBH I was not a fan of `$bind_var` in case of `ext/pdo`. Of course, we shouldn't differ from what PDO uses (alternatively, we can change both ^^).

Reply:
We can call it `$variable`. 
I agree `$bind_var` or `$bindVar` looks nasty. 
Other options would be `$reference`, `$binding`, `$boundVariable` etc.

Reply:
My logic for PDO was that, when used in named params, it ended up reading better:

```php
$db->bindParam(
param: 2,
bind_var: $myparam
);
```

That is, "bind param 2 to the variable $myparam".

Reply:
How about this? 

```
$db->bindParam(
  id: 2,
  variable: $myparam
);
```

Reply:
In line with #6272, this should be `$var` now.